### PR TITLE
fix: ui

### PR DESCRIPTION
**Pull request by alexandrebodin**
### What does it do?

Fixes the following issues

- log user out on unauthorized requests
- Broken icons in users & permissions
- missing confirm modal on user delete
- app logo customization broken
- RBAC permissions matrix broken
- Infinite loading because of unecessary i18N initializer
- Rbac conditions modal broken
- Fix menu not showing EE links before refresh

### CODE REVIEW

File: packages/core/admin/admin/src/pages/Settings/pages/Roles/components/ContentTypeCollapses.tsx [hunk 1]
```diff
@@ -119,6 +122,7 @@ const Collapse = ({
   const { formatMessage } = useIntl();
   const { modifiedData, onChangeParentCheckbox, onChangeSimpleCheckbox } =
     usePermissionsDataManager();
+  const [isConditionMoalOpen, setIsConditionModalOpen] = React.useState(false);
```
Comment (remidej) [inline packages/core/admin/admin/src/pages/Settings/pages/Roles/components/ContentTypeCollapses.tsx:125]:
```suggestion
  const [isConditionModalOpen, setIsConditionModalOpen] = React.useState(false);
```

File: packages/core/admin/admin/src/pages/Settings/pages/Roles/components/PluginsAndSettings.tsx [hunk 2]
```diff
@@ -144,6 +144,7 @@ const SubCategory = ({
 }: SubCategoryProps) => {
   const { modifiedData, onChangeParentCheckbox, onChangeSimpleCheckbox } =
     usePermissionsDataManager();
+  const [isConditionMoalOpen, setIsConditionModalOpen] = React.useState(false);
```
Comment (remidej) [inline packages/core/admin/admin/src/pages/Settings/pages/Roles/components/PluginsAndSettings.tsx:147]:
```suggestion
  const [isConditionModalOpen, setIsConditionModalOpen] = React.useState(false);
```

File: packages/core/admin/admin/src/reducer.ts [hunk 3]
```diff
@@ -50,19 +68,34 @@ const adminSlice = createSlice({
       window.localStorage.setItem(LANGUAGE_LOCAL_STORAGE_KEY, action.payload);
       document.documentElement.setAttribute('lang', action.payload);
     },
+    setToken(state, action) {
```
Comment (remidej) [inline packages/core/admin/admin/src/reducer.ts:71]: should we type the actions?

File: packages/core/admin/admin/src/pages/Settings/pages/Roles/components/CollapsePropertyMatrix.tsx [hunk 4]
```diff
@@ -225,7 +224,7 @@ const ActionRow = ({
                           },
                         });
                       }}
-                      value={checkboxValue}
+                      checked={checkboxValue}
```
[the remark this answers is not in the document]
Comment (alexandrebodin) [reply packages/core/admin/admin/src/pages/Settings/pages/Roles/components/CollapsePropertyMatrix.tsx:227]: Storybook is outdated most likely

File: packages/core/admin/admin/src/reducer.ts (hunk 3 above)
Comment (alexandrebodin) [reply packages/core/admin/admin/src/reducer.ts:71]: yes

**Comment by alexandrebodin** [comment]
> Could you please add links to bugs to validate/QA.

We don't have tickets they were just in the last blitz doc

**Review by remidej — APPROVED** [review]
LGTM, couldn't break any of the mentioned fixes

File: packages/core/review-workflows/admin/src/routes/settings/components/Stages.tsx [hunk 5]
```diff
@@ -517,25 +517,34 @@ const PermissionsField = ({ disabled, name, placeholder, required }: Permissions
...
-    if (!isLoading && roles.length === 0) {
+    if (
+      !isLoading &&
+      getRolesError &&
+      'status' in getRolesError &&
+      getRolesError.status == 403 &&
+      rolesErrorCount.current === 0
+    ) {
+      rolesErrorCount.current = 1;
+
```
Comment (remidej) [inline packages/core/review-workflows/admin/src/routes/settings/components/Stages.tsx:537]: @alexandrebodin I'm not able to reproduce that last fix

https://github.com/strapi/strapi/assets/8087692/0470c565-32e1-4f2c-b55d-37f8842602d0

Comment (alexandrebodin) [reply packages/core/review-workflows/admin/src/routes/settings/components/Stages.tsx:537]: if you remove the permissions to read roles in your test role you should get it

**Review by remidej — APPROVED** [review]

**Pull request merged**